modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -72,22 +154,23 @@ $ oc adm release extract \
 endif::aws-sts[]
 ifdef::google-cloud-platform[]
 ----
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
```

Comment:
This caught my eye while closing tabs - only one command per block, so if you want to set this variable it needs to be added as a step above this one.

Reply:
@jeana-redhat Thank you! I've made the change, much appreciated!

Reply:
- Right now, lines 156-158 & 163 are in the GCP content. I believe they belong in the Alibaba content.
- While the `$ RELEASE_IMAGE=$` command now has its own block, it still needs to be an actual distinct step (one command per step). Something like this:
```
. Set the `$RELEASE_IMAGE` variable by running the following command:
+
[source,terminal]
----

----

. Extract the list of `CredentialsRequest` objects from the {product-title} release image by running the following command:
+
[source,terminal]
----
$ oc adm release extract \
--credentials-requests \
--cloud=alibabacloud \
--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
--from=$RELEASE_IMAGE
----